src/my_app/assets/env/.env.development

```
@@ -1 +1,2 @@
 ENV='Development'
+API_KEY='<DEV_API_KEY>'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)